src/BinaryParsers/ProgramDatabase/Pdb.cs

```diff
@@ -92,34 +92,44 @@ private void Init(string peOrPdbPath, string symbolPath)
         {
             try
             {
-                DiaSource diaSource = new DiaSource();
-                Environment.SetEnvironmentVariable("_NT_SYMBOL_PATH", "");
+                PlatformSpecificHelpers.ThrowIfNotOnWindows();
+                WindowsNativeLoadPdbUsingDia(peOrPdbPath, symbolPath);
+            }
+            catch (PlatformNotSupportedException ex)
```

Comment:
I wonder whether we wouldn't want to let this specific exception through as it is. It is one thing for PDB loading to be entirely unsupported. Generally, a consumer should simply turn of broadstroke functionality as a result. A PDB load issue should indicate that we had a chance to load the PDB but it failed for some reason (e.g., the PDB could not be located). This distinction is helpful (you have a chance to succeed but it went wrong somehow vs. you have no chance at all). Interested in your thoughts.